quarkus/tests/integration/src/main/java/org/keycloak/it/utils/DockerKeycloakDistribution.java

```diff
@@ -58,50 +72,75 @@ private GenericContainer getKeycloakContainer() {
         }
         fetchDockerfile();
         return new GenericContainer(
-                new ImageFromDockerfile()
+                new ImageFromDockerfile("keycloak.x-under-test", false)
                         .withFileFromFile("keycloakx.tar.gz", distributionFile)
                         .withFileFromFile("Dockerfile", cachedDockerfile)
                         .withBuildArg("KEYCLOAK_DIST", "keycloakx.tar.gz")
         )
-                .withExposedPorts(8080);
-    }
-
-    public <T> DockerKeycloakDistribution(boolean debug, boolean manualStop, boolean reCreate) {
-        this.debug = debug;
-        this.manualStop = manualStop;
+                .withExposedPorts(8080)
+                .withStartupAttempts(1)
+                .withStartupTimeout(Duration.ofSeconds(120))
```

Comment:
I increased it on purpose since the augmentation is, sometimes, taking quite a lot of time. 60 seconds was pretty close and failing sporadically, I think 120 is a number we won't change any time soon.